cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverTest.java

```diff
@@ -108,7 +109,7 @@ public static void startLogSaver() throws Exception {
     cConf.set(LoggingConfiguration.LOG_BASE_DIR, logBaseDir);
 
     injector = KAFKA_TESTER.getInjector();
-    impersonator = injector.getInstance(Impersonator.class);
+    impersonator = injector.getInstance(DefaultImpersonator.class);
```

Comment:
we're still not binding Impersonator to anything, and still getting the concrete class.
Wasn't this causing issues?